content/en/integrations/guide/oracle-check-upgrade-7.53.0.md

```diff
@@ -0,0 +1,231 @@
+---
+title: Oracle Integration in the Agent 7.53.0
+kind: guide
+---
+
+# What has changed?
+
+- The Oracle integration is completely rewritten in the Agent release `7.53.0`.
+
+- The new check is easier to set up, and is less prone to configuration errors because it doesn't require any external clients such as Oracle JDBC drivers. The `jdbc` related parameters are ignored in the new check.
+
+- The new Oracle check emits the same metrics as the deprecated Oracle integration.
+
+- System metrics in multi-tenant databases are emitted for each pluggable database.
+
+- The metrics are emitted with the database host name as recorded in the system view `v$instance.host_name` instead of the Agent host name. That can break dashboards and monitors which are based on the host name tag.
```

Comment:
Remove "That can break dashboards and monitors which are based on the host name tag.".

Reply:
Can keep a note like that in an internal guide, but I don't think it makes sense to include in this type of public-facing doc.